examples/MiscExamples.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Spark.CSharp.Core;
+using NUnit.Framework;
+
+namespace Microsoft.Spark.CSharp.Examples
+{
+    class MiscExamples
+    {
+        /// <summary>
+        /// Example to compute th evalue of Pi
+        /// </summary>
+        [Example]
+        internal static void PiExample()
+        {
+            const int slices = 2;
```

Comment:
This will likely suffer from the same test precision problems that PR #183 hit on Travis CI build.

Suggest using `slices = 3` might be less brittle.